### DISCUSSION TITLE
UI icons

### DISCUSSION THREAD

**Discussion by ghost** [xournalpp/xournalpp#4361] [Q&A] [answered]
hi. i'm looking for the icons for  _"find", "go to last page", "go to first page"_, etc. i've searched in _Shared resources folder_ and _ui_ subfolder but they are not there. I mean, there are only a few ones missing but i can´t find them... Thanks :)

**Comment by rolandlo** [comment, marked as answer]
Those are stock icons, not custom icons.
- "go to last page" -> "go-last"
- "go to first page" -> "go-first"
- "find" -> (probably) "edit-find" (which icon do you mean exactly)?

They are looked for somewhere in `usr/share/icons` (under Linux)

See [`ToolMenuHandler.cpp`](https://github.com/xournalpp/xournalpp/blob/811ab29cec4ae1033a48047f0e01c8fa46d6dec0/src/core/gui/toolbarMenubar/ToolMenuHandler.cpp#L341-L573) and [`main.glade`](https://github.com/xournalpp/xournalpp/blob/master/ui/main.glade) for further information.

**Reply by ghost** [reply]
thanks :)
